internal/tests/storage/profile_datasource_test.go

```diff
@@ -0,0 +1,48 @@
+package tests
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+
+	tests "github.com/orange-cloudavenue/terraform-provider-cloudavenue/internal/tests/common"
+)
+
+//go:generate go run github.com/FrangipaneTeam/tf-doc-extractor@latest -filename $GOFILE -example-dir ../../../examples -test
+const testAccProfileDataSourceConfig = `
+data "cloudavenue_storage_profile" "example" {
+	name = "gold"
+}
+`
+
+func TestAccProfileDataSource(t *testing.T) {
+	dataSourceName := "data.cloudavenue_storage_profile.example"
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { tests.TestAccPreCheck(t) },
+		ProtoV6ProviderFactories: tests.TestAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Read testing
+			{
+				// Apply test
+				Config: testAccProfileDataSourceConfig,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttrSet(dataSourceName, "id"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "vdc"),
+					resource.TestCheckResourceAttrSet(dataSourceName, "name"),
```

Comment:
change this line by: 
resource.TestCheckResourceAttr(dataSourceName, "name", "gold"),